modules/lib.lua

```diff
@@ -294,6 +294,7 @@ else
 
     ---@class LibDrawText3DParams : LibDrawTextParams
     ---@field coords vector3
+    ---@field disableDrawRect? boolean default: `true`
```

Comment:
```suggestion
    ---@field disableDrawRect? boolean
```
This can be removed now